pkgs/development/python-modules/pylibftdi/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  disabled = pythonOlder "3"; # compatibility with httpserver
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "132c44rs0f2zm22hnhg93p47aaw6llpfzavc64dcqrn8719mlqfw";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    libusb1
+  ];
+
+  # TODO Several modules have been renamed for python3. ReadTimeout
+  # doesn't appear to be part of pylibftdi at all... These changes
+  # should be upstreamed.
+  patches = [
+    ./make-python3-compatible.patch
```

Comment:
Please use `fetchpatch` to pull the upstream patch from https://bitbucket.org/codedstructure/pylibftdi/commits/3afa277f54d0585945e68d0a7b9a32d7b7c7e7d4/raw
Or bump to version 1.18.1 which supports python 3 [1].

[1] https://pypi.org/project/pylibftdi/